include/camp/defines.hpp

```diff
@@ -88,6 +88,13 @@ namespace camp
 
 #define CAMP_SUPPRESS_HD_WARN
 
+#elif defined(SYCL_LANGUAGE_VERSION)
+#define CAMP_HAVE_SYCL 1
+#define CAMP_DEVICE
+#define CAMP_HOST_DEVICE
+#pragma warn("Sycl compiler!")
```

Comment:
Do you mean to leave this in?

Reply:
No, I'd forgotten to remove it.  I'll remove it now.